attest/verifier/types/src/verification.rs

```diff
@@ -270,13 +272,13 @@ impl Message for VerificationSignature {
 }
 
 /// Structure for holding the contents of the Enclave's Report Data.
-/// The Enclave's ReportData member contains a SHA256 hash of this structure's
-/// contents.
+/// The Enclave Quote's ReportData member contains a SHA256 hash of this
+/// structure's contents.
 #[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq)]
 pub struct EnclaveReportDataContents {
     nonce: QuoteNonce,
     key: X25519Public,
-    custom_identity: [u8; 32],
+    custom_identity: Option<[u8; 32]>,
```

Comment:
I updated this to an option as it's only actually used by the fog enclaves, the other ones currently use a "null" identity that's all zeros, but we could change this in the future to be clearer that there is no custom id.